haas/cli.py

```diff
@@ -166,25 +172,58 @@ def user_create(username, password, is_admin):
 
 @cmd
 def network_create(network, creator, access, net_id):
-    """Create a link-layer <network>.  See docs/networks.md for details"""
-    url = object_url('network', network)
-    do_put(url, data={'creator': creator,
-                      'access': access,
-                      'net_id': net_id})
+    """Create a link-layer <network>.  See docs/networks.md for details
+            --          --              --              --
+        network: Any string to name the network
```

Comment:
We don't need to duplicate the documentation from docs/networks.md or docs/rest_api.md here, in accordance with [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).